extension/wrapper.cpp

```diff
@@ -116,6 +116,30 @@ Satrec_twoline2rv(PyTypeObject *cls, PyObject *args)
     return (PyObject*) self;
 }
 
+static PyObject *
+Satrec_sgp4init(PyTypeObject *cls, PyObject *args)
+{
+    // char *string1, opsmode[1];
```

Comment:
You can remove these comments when you're done with the code inside.

Reply:
Removed.